toolchain/gcc/common.mk

```diff
@@ -128,10 +128,6 @@ ifneq ($(CONFIG_mips)$(CONFIG_mipsel),)
   GCC_CONFIGURE += --with-mips-plt
 endif
 
-ifndef GCC_VERSION_4_8
```

Comment:
It means you should add the line below unconditionally instead of removing all of them.

Reply:
Correct, managed to commit old patch. Thanks